setup.py

```diff
@@ -14,7 +14,7 @@
         'pytest',
         'pytest-cov',
     ],
-    ':python_version < "3.4"': [
+    'python_version < "3.4"': [
```

Comment:
```$ easy_install --version
setuptools 18.3.1 from /usr/local/lib/python2.7/site-packages (Python 2.7)```
Not the latest, but certainly not super old—PyPI says that that version's from September 2015.